config/Rules.am

```diff
@@ -8,6 +8,7 @@ DEFAULT_INCLUDES = -include ${top_builddir}/zfs_config.h
 AM_LIBTOOLFLAGS = --silent
 
 AM_CFLAGS  = -std=gnu99 -Wall -Wstrict-prototypes -fno-strict-aliasing
+AM_CFLAGS += @NO_OMIT_FRAME_POINTER@
```

Comment:
nit: I believe you want `$(NO_OMIT_FRAME_POINTER)` instead of `@NO_OMIT_FRAME_POINTER@`.

Reply:
both seem to work, but I agree $() is better.
fixed.